types/testing.go

```diff
@@ -63,6 +63,14 @@ func NewMockSigner(kis []KeyInfo) MockSigner {
 	return ms
 }
 
+// NewMockSignersAndKeyInfo is a convenience function to generate a given number of (mock)
```

Comment:
Was also seeing this setup all over the place, and it just made sense to me to abstract it.  Sometimes we use the resulting KeyInfos and sometimes we don't.